static/styles/template.css

```diff
@@ -95,3 +201,135 @@ header{
     font-size: 15px;
     text-decoration: none;
   }
+
+
+  /* noscript{
+    display: none;
+} */
+
+.noscript-container{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+}
+
```

Comment:
I did not know about this, interesting technique!

Reply:
u def should try this game https://flexboxfroggy.com/ for better understanding FLEX!

Reply:
Saved this link, thanks